services/src/main/scala/cromwell/services/metadata/impl/archiver/ArchiveMetadataSchedulerActor.scala

```diff
@@ -130,11 +130,11 @@ class ArchiveMetadataSchedulerActor(archiveMetadataConfig: ArchiveMetadataConfig
     }
   }
 
-  private def getGcsPathForMetadata(workflow: WorkflowQueryResult): Future[Path] =  {
+  private def getGcsPathForMetadata(workflow: WorkflowQueryResult): Path =  {
     val bucket = archiveMetadataConfig.bucket
     val workflowId = workflow.id
     val rootWorkflowId = workflow.rootWorkflowId.getOrElse(workflowId)
-    Future(PathFactory.buildPath(s"gs://$bucket/$rootWorkflowId/$workflowId.csv", archiveMetadataConfig.pathBuilders))
+    PathFactory.buildPath(s"gs://$bucket/$rootWorkflowId/$workflowId.csv", archiveMetadataConfig.pathBuilders)
```

Comment:
Assuming this never throws exceptions, this is fine

Reply:
It looks like `buildPath` can throw [PathParsingException](https://github.com/broadinstitute/cromwell/blob/29b929e9963c3220e4bdf6745589448dfcd6fc45/core/src/main/scala/cromwell/core/path/PathFactory.scala#L80-L85)

Reply:
Isn't it fine even if it does? The usage is inside a `Future` chain, so I think it would get correctly propagated as a failed `Future` there.

Reply:
But won't this change take it out of a `Future` chain?